zppy/templates/amwg.csh

```diff
@@ -436,7 +436,7 @@ set sig_lvl = 0.05           # level of significance
 {% if machine == 'compy' %}
 #setenv DIAG_HOME /compyfs/software/amwg/amwg_diag
 setenv DIAG_HOME /compyfs/diagnostics/amwg/amwg_diag
-{% elif machine == 'cori' %}
+{% elif machine in ['pm-cpu', 'pm-gpu'] %}
 #setenv DIAG_HOME /global/cfs/cdirs/e3sm/golaz/amwg/amwg_diag
```

Comment:
Take this line out?

Reply:
The commented line?

Reply:
Yes.  No useful to anyone but @golaz, right?